docs/guides/admin/docs/upgrade.md

```diff
@@ -67,6 +67,7 @@ Configuration Changes
 - `etc/org.opencastproject.scheduler.impl.SchedulerServiceImpl.cfg` has a new option `maintenance` which temporarily
   disables the scheduler if set to `true`.
 
+
```

Comment:
@mliradelc & @lkiesow although it would be desirable to not add a new line to upgrade.md...